scripts/qa_crowbarsetup.sh

```diff
@@ -1662,14 +1662,7 @@ EOF
         service smb restart
     fi
 
-    sleep 20
-    if ! curl -m 59 -s $crowbar_api  > /dev/null || \
-        ! curl -m 59 -s --digest --user crowbar:crowbar $crowbar_api  | \
```

Comment:
this drops the part that uses digest auth so we might only learn about problems there later